src/testers.jl

```diff
@@ -172,7 +172,7 @@ function test_rrule(
     config::RuleConfig,
     f,
     args...;
-    output_tangent=Auto(),
+    output_cotangent=Auto(),
```

Comment:
So I don't forget if someone does come back to this.
This can't be done.
It is a breaking change.

In general we have decide to use `tangent` as the word to refer to both `tangent`s and `cotangent`s